src/typescript/azure-openapi-validator/rules/PathParameterMustNotBeEmpty.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+export const PathParametersMustNotBeEmpty: string = "PathParametersMustNotBeEmpty";
+
+rules.push({
+  id: "R2067",
+  name: PathParametersMustNotBeEmpty,
+  severity: "error",
```

Comment:
I am not convinced that we should flag minLength missing for parameters as an error as of yet, might light up the validation charts wildly, please verify with kirthi

Reply:
It is an error, but I see your point as this will probably blow up the error count from what I saw in the specs. Either way, I will work to fix the specs. So I would say I'll move it to a warning until we get those fixed and then we can change it to an error

Reply:
Even then, better discuss this with @veronicagg and @kirthik before releasing the rule.